tools/plugins/com.liferay.ide.portlet.core/plugin.xml

```diff
@@ -177,4 +177,15 @@
             id="com.liferay.ide.portlet.core.jsf.jsfLiferayPortletDescriptorHelper">
       </liferayDescriptorHelper>
    </extension>
+   <extension
+         point="com.liferay.ide.project.core.portletFrameworks">
+      <portletFramework
+            class="com.liferay.ide.portlet.core.spring.SpringPortletFramework"
+            description="The Spring MVC portlet that run inside of Lifreay Portlet"
```

Comment:
Also lets add a help url that points here: http://docs.spring.io/autorepo/docs/spring/3.2.x/spring-framework-reference/html/portlet.html